freedomofpress/securedrop-https-everywhere-ruleset
# remove test version, add release key signed securedrop HTTPS everywhere ruleset channel

Towards https://github.com/freedomofpress/securedrop/issues/3668

Few things to note: 
- I've used the official SecureDrop release key for signing these rules
- First org onboarded is Lucy Parsons Labs
- This is using option A from the design doc I shared last week (I can change this in review since now that I have the process down doing a ruleset release is fast, but now is the last chance to change without disruption to users, let's check in tomorrow on this)

# Testing

For easy testing/review of this PR I deployed these changes to a personal page [here](https://redshiftzero.github.io/securedrop-httpse/) (I was going to wait for merge into master here so we could use staging securedrop server but... I think it's useful to have a review/test step first). 

To add the ruleset update channel:

1. Open Tor Browser 
2. Type `about:addons` in the URL bar
3. Click "HTTPS Everywhere"
4. Click "Preferences"
5. Click "Update Channels"
6. Type "SecureDrop” and click "Add Update Channel"
7. In JWK section, add the content of `release-pubkey.jwk` in this branch (this is just the JWK version of our existing RSA release public key)
8. In path prefix, add https://redshiftzero.github.io/securedrop-httpse/
9. Click update
10. If all goes well, you should see "Stored rulesets version: 2020.4.14" appear
11. Load `lucyparsonslabs.com.securedrop.tor.onion`
12. Tor Browser should redirect you to `http://qn4qfeeslglmwxgb.onion/`
